lib/livebook/intellisense.ex

```diff
@@ -451,8 +455,8 @@ defmodule Livebook.Intellisense do
          context
        ) do
     join_with_divider([
+      format_definition_link(module, context, {:module, module}),
```

Comment:
Oh, let's the order as is, the screenshot I shared is something else. I am thinking we could not include the link in the docs, but add it on the client, but I can explore that separately :)